elements/pfe-accordion/demo/index.html

```diff
@@ -15,7 +15,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap-reboot.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/typebase.css/0.5.0/typebase.css">
     -->
-    
+
```

Comment:
@chrisdo1 sorry about the whitespaces & changes in this file. I tried to undo them, VScode is not cooperating